src/components/Book.js

```diff
@@ -1,47 +1,65 @@
-import React from 'react';
+import React, { useState } from 'react';
 import PropTypes from 'prop-types';
 import { useDispatch, useSelector } from 'react-redux';
+import { CircularProgressbar } from 'react-circular-progressbar';
 import { removeBookItem } from '../redux/books/booksSlice';
+import 'react-circular-progressbar/dist/styles.css';
 
 const Book = ({
   id, title, author, category,
 }) => {
   const dispatch = useDispatch();
+  const [percentage, setPercentage] = useState(64);
   const book = useSelector((state) => state.books.books.find((book) => book.item_id === id));
 
   if (!book) {
     return null;
   }
 
+  const handleProgress = () => {
+    let newPercentage = percentage;
+    if (percentage < 100) {
+      newPercentage += 2;
+    }
+    setPercentage(newPercentage);
+  };
+
   return (
     <li>
       <div className="Book">
-        <p>{category}</p>
-        <h3>{title}</h3>
-        <p>{author}</p>
-        <div>
-          <button type="button">comment</button>
+        <span className="category">{category}</span>
+        <h3 className="Title">{title}</h3>
+        <span className="Author">{author}</span>
+        <div className="book-btn">
+          <button type="button">Comment</button>
```

Comment:
- [ ] I think it would be great if you have the same colors as given design since the color in category that above the book name and the placholder for update progress have different colors than yours so kindly make them the same colors as given design 🚀😊💯🔥🤩

## I refered to you in the image below ⬇
![image](https://user-images.githubusercontent.com/58553711/230515438-191760a2-0133-46a5-9765-dd744ac8982f.png)

- Zeplin design
![image](https://user-images.githubusercontent.com/58553711/230515517-82a34474-54e8-4372-bd9c-c99e2f32cfc3.png)

Reply:
Changes effected, thanks